### DISCUSSION TITLE
Does signing in to websites slow down BetterFox?

### DISCUSSION THREAD

**Discussion by GrayLanterns** [yokoffing/Betterfox#222] [Q&A]
I have only recently tweaked my native Firefox with BF, and the improvements I'm noticing are quite significant. Now I don't wanna do something nasty which could potentially take away the current speed of browsing. Even if that includes signing in to websites. Please let me know if it makes any difference.

**Comment by GrayLanterns** [comment]
Maybe you forgot to drop by here @yokoffing I had a query if you could kindly look into it.

**Comment by yokoffing** [comment]
Not an issue...

**Reply by GrayLanterns** [reply]
I was a little skeptical about the premise, thanks for letting me know. 👍
